packages/core/src/shared/resourcefetcher/node/httpResourceFetcher.ts

```diff
@@ -0,0 +1,129 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as fs from 'fs' // eslint-disable-line no-restricted-imports
+import * as http from 'http'
+import * as https from 'https'
+import * as stream from 'stream'
+import got, { RequestError } from 'got'
+import urlToOptions from 'got/dist/source/core/utils/url-to-options'
+import Request from 'got/dist/source/core'
+import { VSCODE_EXTENSION_ID } from '../../extensions'
+import { getLogger, Logger } from '../../logger'
+import { Timeout, CancellationError, CancelEvent } from '../../utilities/timeoutUtils'
+import { isCloud9 } from '../../extensionUtilities'
+import { Headers } from 'got/dist/source/core'
+
+// XXX: patched Got module for compatability with older VS Code versions (e.g. Cloud9)
+// `got` has also deprecated `urlToOptions`
+const patchedGot = got.extend({
```

Comment:
patchedGot is removed in max's pr https://github.com/aws/aws-toolkit-vscode/pull/6376